### PR TITLE
Fix: Token List click on share button takes user to next screen

### PR DESCRIPTION
**Pull request by prakash-bhatt-rs**

### CODE REVIEW

**Comment by prakash-bhatt-rs** [comment]
- I have extracted the TokenTile as a separate component, 
- Fixed click error (In wallet screen upon clicking the Token share button, page was navigating automatically to token info screen ).
- Used TokenTile component into Token at profile.
- Added airdrop-amt as query param for Token share url.

**Comment by harshitasrivastav28** [comment]
When I click on the link shared to other users, the app throws my Coffee break error. @prakash-bhatt-rs

**Comment by prakash-bhatt-rs** [comment]
web::share_url() API is already appending the base url on share. This was causing error like base appearing twice on the shared url. 
e.g. 
https://yral.com/yral.com/profile/j2zob-vt6qd-fgoz2-q2hxj-huvmo-nl7jl-rzcm7-ikvds-hhkfj-rllw3-lqe?tab=tokens

**Comment by harshitasrivastav28** [comment]
The issue has been resolved and working as expected. @prakash-bhatt-rs This is good to go! Please assign it to reviewer

File: ssr/src/page/profile/tokens.rs [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
     utils::token::{get_token_metadata, TokenCans},
 };
 
-#[component]
-fn TokenViewFallback() -> impl IntoView {
-    view! {
-        <div class="w-full h-20 rounded-xl border-2 border-neutral-700 bg-white/15 animate-pulse"></div>
-    }
-}
+// #[component]
```
Comment (rupansh-gob) [inline ssr/src/page/profile/tokens.rs:14]: Nuke the unused component instead of commenting it

File: ssr/src/component/share_popup.rs [hunk 2]
```diff
@@ -107,7 +110,12 @@ fn SocialShare(share_link: String, message: String) -> impl IntoView {
 }
 
 #[component]
-pub fn ShareButtonWithFallbackPopup(share_link: String, message: String) -> impl IntoView {
+pub fn ShareButtonWithFallbackPopup(
+    share_link: String,
+    message: String,
+    style: Option<String>,
```
Comment (rupansh-gob) [inline ssr/src/component/share_popup.rs:116]: use 
```rust
#[prop(optional)] style: String
```

**Review by rupansh-gob — CHANGES_REQUESTED** [review]
minor changes

**Review by rupansh-gob — APPROVED** [review]

**Pull request merged**